tests/__mocks__/@ubiquity-os/ubiquity-os-kernel.ts

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  CONFIG_FULL_PATH: ".github/.ubiquity-os.config.yml",
+  DEV_CONFIG_FULL_PATH: ".github/.ubiquity-os.config.dev.yml",
+} as jest.Mocked<typeof import("@ubiquity-os/ubiquity-os-kernel")>;
```

Comment:
Is this needed now?

Reply:
![image](https://github.com/user-attachments/assets/eff11022-0217-44ca-8084-66cdb3fe4683)

Yeah Jest still complaining